js/script.js

```diff
@@ -6,56 +6,54 @@ let newBook; // variable for book object that's pushed in array
 let title;
```

Comment:
- Line 4 and 5 of your `script.js` you use an array and object variables outside a class. The project requirement mentions: "You can use objects and arrays, but do it inside classes." Kindly create a class for a new book and store the booklist inside a class too.